main_doc/styles.scss

```diff
@@ -0,0 +1,453 @@
+                    font-weight: 400;
+                    line-height: 20px;
+                }
+            }
+        }
+        .work-button {
+            text-align: center;
+        }
+    }
+}
+
+.testymonials {
+    height: 840px;
+    background-image: url(../img/Mask_Group_brown.svg);
+    background-size: auto;
+    background-position: center;
+    background-color: $main-color; 
+
+    .content-testymonials {
+        height: 840px;
+        width: 100%;
+        display: flex;
+        justify-content: space-evenly;
+        flex-wrap: wrap;
+        align-content: center;
+        flex-direction: column;
+
+        h2 {
+            color: $third-color;
+            font-family: Sen, sans-serif;
+            font-size: 144px;
+            font-weight: 800;
+            line-height: 96%;
+            text-transform: uppercase;
+            text-align: center;
+        }
+
+        .type-content-testymonials {
+            display: flex;
+            justify-content: space-evenly;
+            flex-wrap: wrap;
+            align-content: center;
+
+            .type-testymonials {
+                width: 80%;
+
+                .testymonials-card {
+                    border-radius: 25px;
+                    border-color: $third-color;
+                    padding: 5px 20px;
+
+                    .testymonials-name {
+                        color: $main-color;
+                        font-size: 16px;
+                        font-weight: 900;
+                        line-height: 24px;
+                    }
+                    .testymonials-text {
+                        font-size: 14px;
+                        font-weight: 400;
+                        line-height: 20px;
+                    }
+                }  
+            }
+        }
+        .testymonials-button {
+            text-align: center;
+        }
+    }
+}
+footer {
+    height: 671px;
+    position: relative;
+
+    .contuct-us-filling {
+        height: 480px;
+        width: 100%;
+        display: flex;
+        justify-content: space-evenly;
+        flex-wrap: wrap;
+        align-content: center;
+        flex-direction: column;
+
+        h2 {
+            color: $main-color;
+            font-family: Sen, sans-serif;
+            font-size: 144px;
+            font-weight: 800;
+            line-height: 96%;
+            text-transform: uppercase;
+            text-align: center;  
+        }
+
+        .contuct-us-form {
+            width: 70%;
+            display: flex;
+            justify-content: flex-end;
+            flex-wrap: wrap;
+
+            form {
+                height: 40%;
+
+                .form-content-meaning {
+                    display: flex;
+                    flex-wrap: wrap;
+                    align-content: space-around;
+                    height: 90%;
+                    
+                    .form-input-meaning {
+                        border-radius: 61px;
+                        background: #FFD39F;

```

Comment:
I see this color is used several times in the code, so it should be moved to a variable